recipes/pdal-parallelizer/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "pdal-parallelizer" %}
+{% set version = "1.10.19" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9dc874148c437f42c8df5ee6df4b1a8235681ef55616b6c43e846def6701f00b
+
+build:
+  number: 0
```

This is pure Python and can be marked as a "noarch" build:
```suggestion
  number: 0
  noarch: python
```